code/modules/food&drinks/kitchen machinery/processor.dm

```diff
@@ -34,6 +34,15 @@
 	var/output
 	var/time = 40
 /datum/food_processor_process/proc/process_food(loc, what, var/obj/machinery/processor/processor)
+	if(istype(what, /obj/item/weapon/fish))
```

Comment:
Alright, see this line? See it? Are you sure that ever fucking time I process a piece of corn, that I should make the food processor make sure its not a fucking fish?